doc/lsst.pipe.tasks/index.rst

```diff
@@ -12,3 +12,42 @@ Python API reference
 ====================
 
 .. automodapi:: lsst.pipe.tasks.assembleCoadd
+.. automodapi:: lsst.pipe.tasks.calibrate
+.. automodapi:: lsst.pipe.tasks.characterizeImage
+.. automodapi:: lsst.pipe.tasks.coaddBase
+.. automodapi:: lsst.pipe.tasks.coaddHelpers
+.. automodapi:: lsst.pipe.tasks.coaddInputRecorder
+.. automodapi:: lsst.pipe.tasks.colorterms
+.. automodapi:: lsst.pipe.tasks.dcrAssembleCoadd
+.. automodapi:: lsst.pipe.tasks.dcrMultiBand
+.. automodapi:: lsst.pipe.tasks.exampleCmdLineTask
+.. automodapi:: lsst.pipe.tasks.exampleStatsTasks
+.. automodapi:: lsst.pipe.tasks.fakes
+.. automodapi:: lsst.pipe.tasks.getRepositoryData
+.. automodapi:: lsst.pipe.tasks.imageDifference
+.. automodapi:: lsst.pipe.tasks.ingestCalibs
+.. automodapi:: lsst.pipe.tasks.ingestPgsql
+.. automodapi:: lsst.pipe.tasks.ingest
+.. automodapi:: lsst.pipe.tasks.interpImage
+.. automodapi:: lsst.pipe.tasks.makeCoaddTempExp
+.. automodapi:: lsst.pipe.tasks.makeDiscreteSkyMap
+.. automodapi:: lsst.pipe.tasks.makeSkyMap
+.. automodapi:: lsst.pipe.tasks.matchBackgrounds
+.. automodapi:: lsst.pipe.tasks.measurePsf
+.. automodapi:: lsst.pipe.tasks.multiBand
+.. automodapi:: lsst.pipe.tasks.photoCal
+.. automodapi:: lsst.pipe.tasks.processCcd
+.. automodapi:: lsst.pipe.tasks.propagateVisitFlags
+.. automodapi:: lsst.pipe.tasks.registerImage
+.. automodapi:: lsst.pipe.tasks.repair
+.. automodapi:: lsst.pipe.tasks.repositoryIterator
+.. automodapi:: lsst.pipe.tasks.scaleVariance
+.. automodapi:: lsst.pipe.tasks.scaleZeroPoint
+.. automodapi:: lsst.pipe.tasks.selectImages
+.. automodapi:: lsst.pipe.tasks.setConfigFromEups
+.. automodapi:: lsst.pipe.tasks.setPrimaryFlags
+.. automodapi:: lsst.pipe.tasks.snapCombine
+.. automodapi:: lsst.pipe.tasks.transformMeasurement
+.. automodapi:: lsst.pipe.tasks.version
+.. automodapi:: lsst.pipe.tasks.warpAndPsfMatch
+.. automodapi:: lsst.pipe.tasks.objectMasks
```

Comment:
For the record, @DavidStaker and I discussed this. It won't work because `lsst/pipe/tasks/__init__.py` doesn't import anything useful. There are reasons for that; I suggest that he not try to change it on this ticket.

Reply:
As briefly discussed today with @jdswinbank , this change is to be kept.